lua/tabout/node.lua

```diff
@@ -29,6 +29,8 @@ local get_parent_node = function(node, multi)
     end
 end
 
+local skip_notify = {}
```

Comment:
This would not work, if we do not expose a config option for it.
But I feel like we do not need it, since you can already define for which filetypes you want to run tabout.
And if you expect tabout to be enabled on your filetype you want to see an error message if it does not work 😊